doc/sphinx/source/build/project_build_stm32.rst

```diff
@@ -70,6 +70,10 @@ and generate executable file (.bin/.hex)
                .. image:: /source/build/stm32_exclude_syscalls.png
                   :width: 600
 
+            * Select 'Convert to binary file' to generate a .bin file as a post build output
```

Comment:
Can elaborate this @Shetty-Anush ? Something like (right click on the project->click properties->....)

Reply:
Probably adding a screenshot would help?